src/index.js

```diff
@@ -71,11 +71,11 @@ module.exports = async function main(settings) {
             `Checking widths of image ${color.white(options.selector)}`,
           ),
         )
+        process.stdout.write(
```

Comment:
Why `process.stdout.write` sometimes and `console.log` other times?

Reply:
Well, maybe I only need these two lines, and we can keep `console.log`:

```
process.stdout.clearLine()	
process.stdout.cursorTo(0)
```

Reply:
I tried, it doesn't work, `console.log` seems to add a new line anyway.

Reply:
Ok for this PR, I think we should use a better log API.